### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kickbell**
Hi, 

Fixed some **README.md** and added content **Pulse.md**

### CODE REVIEW

File: Documentation/Contents/Pulse.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+    return self
+  }
+
+  private mutating func riseValueUpdatedCount() {
+    if self.valueUpdatedCount == UInt.max {
+      self.valueUpdatedCount = UInt.min
+    } else {
+      self.valueUpdatedCount += 1
+    }
+  }
```
Comment (tokijh) [inline Documentation/Contents/Pulse.md:131]: This codes updated in https://github.com/ReactorKit/ReactorKit/pull/199.
How about explain with `&+=`?

**Review by tokijh — APPROVED** [review]
Great! Thank you for your contribution!
I saw your post of that [explain of the Pulse](https://kickbell.github.io/ReactorKit-Pulse-EN).
I think you can attach your post directly instead of Pulse.md if you want.

**Comment by kickbell** [comment]
Thanks for the feedback, tokijh. 
Update #199 and Add link in my blog.

**Review by OhKanghoon — APPROVED** [review]
Thanks for your contribution!
I will edit document that need to be modified additionally.

**Pull request merged**